guides/docs/ecto.md

```diff
@@ -447,3 +447,80 @@ In addition to inserts, we can also perform updates and deletes with `Repo.updat
 There is quite a bit more that Ecto can do and we've only barely scratched the surface. With a solid Ecto foundation in place, we're now ready to continue building our app and integrate the web facing application with our backend persistence. Along the way, we'll expand our Ecto knowledge and learn how to properly isolate our web interface from the underlying details of our system. Please take a look at the [Ecto documentation](https://hexdocs.pm/ecto/) for the rest of the story.
 
 In our [context guide](contexts.html), we'll find out how to wrap up our Ecto access and business logic behind modules that group related functionality. We'll see how Phoenix helps us design maintainable applications, and we'll find out about other neat Ecto features along the way.
+
+## Using MySQL
+
+Phoenix applications are configured to use PostgreSQL by default, but what if we want to use MySQL instead? In this guide, we'll walk through changing that default whether we are about to create a new application, or whether we have an existing one configured for PostgreSQL.
+
+If we are about to create a new application, configuring our application to use MySQL is easy. We can simply pass the `--database mysql` flag to `phx.new` and everything will be configured correctly.
+
+```
+$ mix phx.new hello_phoenix --database mysql
+
+```
+This will set up all the correct dependencies and configuration for us automatically. Once we install those dependencies with `mix deps.get`, we'll be ready to begin working with Ecto in our application.
+
+If we have an existing application, all we need to do is switch adapters and make some small configuration changes.
+
+To switch adapters, we need to remove the Postgrex dependency and add a new one for Mariaex instead.
+
+Let's open up our `mix.exs` file and do that now.
+
+```
+defmodule HelloPhoenix.Mixfile do
+  use Mix.Project
+
+  . . .
+  # Specifies your project dependencies.
+  #
+  # Type `mix help deps` for examples and options.
+  defp deps do
+    [
+      {:phoenix, "~> 1.3.0"},
+      {:phoenix_pubsub, "~> 1.0"},
+      {:phoenix_ecto, "~> 3.2"},
+      {:mariaex, ">= 0.0.0"},
```

Comment:
Should this version be locked down, like the others? I would suggest:

```
{:mariaex, "~> 0.8.3"}
```

Reply:
That was what I thought to do initially, but I noticed that creating a project with --database=mysql uses the included dep line.